### PR TITLE
implement beacon validator list ui

### LINKED ISSUE #310: [BN] validator list
- implement validator list on Beacon nodes management screen

Screenshot from 2020-11-17 11-02-57

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #310

### CODE REVIEW

File: src/renderer/ducks/validator/selectors.ts [hunk 1]
```diff
@@ -25,3 +25,18 @@ export const getValidatorBeaconNodes = createSelector(
     getBeaconDictionary,
     (validator, beacons) => validator?.beaconNodes.map((url) => beacons[url]).filter((beacon) => !!beacon) || [],
 );
+
+export const getBeaconNodesValidators = createSelector(getValidators, getValidatorKeys, (validators, keys) => {
```
Comment (morrigan) [inline src/renderer/ducks/validator/selectors.ts:29]: maybe `getValidatorsByBeaconNode` ?

Comment (morrigan) [inline src/renderer/ducks/validator/selectors.ts:29]: define return type for selector?

**Review by morrigan — COMMENTED** [review]
LGTM, left smaller suggestions

File: src/renderer/ducks/validator/selectors.ts (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/ducks/validator/selectors.ts:29]: you mean define types in generics or on anonymous arrow function?

**Review by morrigan — APPROVED** [review]

**Pull request merged**